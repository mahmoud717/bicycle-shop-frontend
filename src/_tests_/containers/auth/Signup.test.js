import React from 'react';

import { shallow } from 'enzyme';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import Signup from '../../../containers/auth/sign_up';

const mockStore = configureMockStore();
const store = mockStore({});
const setup = () => shallow(
  <Provider store={store}>
    <Signup />
  </Provider>,
);

describe('Signup Component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });
  test('renders the Signup Component', () => {
    const page = component.find('.signup');
    expect(page).toMatchSnapshot();
  });
});
