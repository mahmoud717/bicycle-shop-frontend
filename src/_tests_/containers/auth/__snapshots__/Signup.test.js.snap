// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup Component renders the Signup Component 1`] = `ShallowWrapper {}`;
